Add instructions on customizing installation base path for Elastic Agent (#160)
elastic/ingest-docs: docs/en/ingest-management/tab-widgets/install-layout.asciidoc

`/usr/bin/elastic-agent`::
Shell wrapper installed into PATH

> You can install {agent} in a custom base path that's different from `/Library`.  When installing {agent} with the `./elastic-agent install`
> command, provide the custom base path via the `--base-path` CLI option.

Reviewer: ```suggestion
You can install {agent} in a custom base path other than `/Library`.  When installing {agent} with the `./elastic-agent install`
command, use the `--base-path` CLI option to specify the custom base path.
```
Just a small change to avoid `via` (we try not to use Latin expressions to help non-native English speakers and translatability).

`/usr/bin/elastic-agent`::
Shell wrapper installed into PATH

> You can install {agent} in a custom base path that's different from `/opt`.  When installing {agent} with the `./elastic-agent install`
> command, provide the custom base path via the `--base-path` CLI option.

Reviewer: ```suggestion
You can install {agent} in a custom base path other than `/opt`.  When installing {agent} with the `./elastic-agent install`
command, use the `--base-path` CLI option to specify the custom base path.
```

`C:\Program Files\Elastic\Agent\data\elastic-agent-*\logs\default\*-json.log*`::
Log files for {beats} shippers

> You can install {agent} in a custom base path that's different from `C:\Program Files`.  When installing {agent} with the `./elastic-agent install`
> command, provide the custom base path via the `--base-path` CLI option.

Reviewer: ```suggestion
You can install {agent} in a custom base path other than `C:\Program Files`.  When installing {agent} with the `./elastic-agent install`
command, use the `--base-path` CLI option to specify the custom base path.
```
